app/frontend/src/components/forms/TeamManagement.vue

```diff
@@ -97,8 +104,23 @@ export default {
     addNewUsers(users) {
       if (Array.isArray(users) && users.length) {
         users.forEach((user) => {
-          // TODO: Create a new this.tableData object and add it in
-          console.log(user); // eslint-disable-line no-console
+          // if user isnt already in the table
+          if(this.tableData.filter(function(obj) { return obj.userId === user.id; }).length < 1) {
+            // create new object for table row
+            var row = {
```

Comment:
Never use var - this should either be a `let` or a `const`. (In this case, I believe it should be a const).

Reply:
👍